internal/fs/inode/base_dir.go

```diff
@@ -199,8 +199,12 @@ func (d *baseDirInode) CreateChildFile(ctx context.Context, name string) (*Core,
 	return nil, fuse.ENOSYS
 }
 
-func (d *baseDirInode) CreateLocalChildFile(name string) (*Core, error) {
-	return nil, fuse.ENOSYS
+func (d *baseDirInode) InsertFileIntoTypeCache(_ string) {}
+
+func (d *baseDirInode) EraseFromTypeCache(_ string) {}
+
+func (d *baseDirInode) LocalChildFileCore(_ string) Core {
+	return Core{}
```

Comment:
Returning empty core is not good. All the parameters will have default values and becomes difficult to debug if something goes wrong.